Gemfile

```diff
@@ -80,6 +80,8 @@ gem "ransack", "~> 2.4" # Searching and sorting
 gem "recaptcha", "~> 5.6", require: "recaptcha/rails" # Helpers for the reCAPTCHA API
 gem "redcarpet", "~> 3.5" # A fast, safe and extensible Markdown to (X)HTML parser
 gem "redis", "~> 4.2.5" # Redis ruby client
+gem "rpush", "~> 5.3" # Push Notification library for Rails
+gem "rpush-redis", "~> 1.1" # Redis module capability for rpush library
```

Comment:
This avoids the need to run rpush migrations (it's a major problem for us because of `strong_migrations`)

Reply:
If the decision to use `rpush-redis` is purely motivated by `strong_migrations` I'd rather try to figure out how to work around them.

Reply:
I can agree that being 100% motivated by the `strong_migrations` problem for this decision is not the best, I definitely could've expanded a bit more:
- These migrations are not trivial and we would have some overhead work to work with them
   - The initial "generate" and then edit them manually to make them compliant with `strong_migrations` ([there's plenty of them](https://github.com/rpush/rpush/tree/master/lib/generators/templates))
   - Over time support each new migration added by rpush and manually edit them for compliance as well (if they're not)
- I think an even stronger argument is that rpush provides a **complete solution**. This means they track each PN sent in a table with a lot of metadata that we may not be interested in keeping.
   - We have the `notifications` table already, we could simply add some new fields (to monitor which `Notification` triggered a PN delivery for example) instead of rolling out with the "off the shelf" rpush solution in the DB

Reply:
Awesome, thanks for the context, that makes a lot more sense now.